pandas/io/pytables.py

```diff
@@ -943,6 +956,17 @@ def put(self, key, value, format=None, append=False, **kwargs):
         append   : bool, default False
             This will force Table format, append the input data to the
             existing.
+        complevel : int, 0-9, default None
```

Comment:
Can you align the docstring with the signature? Looks like `compilb` and `complevel` are swapped